app/models/user.rb

```diff
@@ -762,7 +766,8 @@ def refresh_counts!
           post_count: Post.for_user(id).count,
           post_deleted_count: Post.for_user(id).deleted.count,
           post_update_count: PostVersion.for_user(id).count,
-          note_count: NoteVersion.where(updater_id: id).count
+          note_count: NoteVersion.for_user(id).count,
+          ticket_count: Ticket.for_user(id).count,
```

Comment:
I see not reason to fix something that isn't broken in the first place. I have not found evidence that the status counter gets out of sync

Reply:
With something that will be counted so infrequently, you are probably correct
I had assumed that *everything* would be included in a count refresh, regardless of how infrequent or unlikely they are to be inaccurate